contracts/BIFI/strategies/Spooky/StrategySpookyV2LP.sol

```diff
@@ -212,37 +210,38 @@ contract StrategySpookyV2LP is StratManager, FeeManager {
     }
 
     function rewardsAvailable() public view returns (uint256, uint256) {
-        uint256 outputBal = ISpookyChefV2(chef).pendingBOO(poolId, address(this));
+        address rewarder = ISpookyChefV2(chef).rewarder(poolId);
```

Comment:
Instead of swapping output and secondOutput here, would it be easier to just deploy with different constructor args?  Other spooky strats have `output = BOO` and `secondOutput = <whatever>`.  If you switch the output/secondOutput and outputToNativeRoute/secondOutputToNativeRoute given to the constructor, it should work as it was?